lib/plugins/aws/lib/naming.test.js

```diff
@@ -811,4 +811,34 @@ describe('#naming()', () => {
       ).to.equal('CustomApiGatewayAccountCloudWatchRole');
     });
   });
+
+  describe('#getCloudFrontDistributionLogicalId()', () => {
+    it('should return CloudFront distribution logical id', () => {
```

Comment:
I wonder whether there's a real value in such unit tests (?)

Technically we write same code twice (it's a bit as writing a function in one file and in other confirming that code of the function matches given string - and that feels counterproductive and not really valuable).

In case of naming I would rather rely on tests of given functionality (which relies on `naming`). Over there via spies we can confirm that given functionality relies on `naming` (but without inspecting what naming function returns)

Reply:
Good point. I agree with the sentiment :+1:

Not sure if we always test for the correct naming generation in the other tests though (what the coverage of the `naming.js` code is if we remove the tests / test file).

Reply:
> Not sure if we always test for the correct naming generation in the other tests though (what the coverage of the naming.js code is if we remove the tests / test file).

Yes, I don't think we should call for any refactor now. But we may just add here special note (for future PR's), and stop writing more tests here, as in most cases they miss the point.